js/player.js

```diff
@@ -33,13 +35,51 @@ class YTShortsPlayer {
     if (!this.video) throw new Error('no video was set to the player');
     if (!this.container) throw new Error('no container was set to the player');
 
-    const playerControls = this.createPlayerControls();
+    this.disableLoopingIfNeeded(this.video);
 
     if (this.container.querySelector('.cfyts-player-controls')) return;
+    const playerControls = this.createPlayerControls();
 
     this.container.appendChild(playerControls);
   }
 
+  disableLoop(videoElement) {
+    setTimeout(() => {
+      videoElement.loop = false;
+    }, 1500);
+  }
+
+  disableLoopingIfNeeded(video) {
+    if (!this.disableInfiniteLoop) return;
+    if (video.isObserved) return;
+    video.isObserved = true;
```

Comment:
Hey thanks for the PR! Unfortunately for me the infinite loop option just works when the page loads, but when toggling the looping option while the video is already loaded it does not change the infinite loop.

for example:
1. video has infinite loop.
2. disable infinite loop
3. video **still** have infinite loop
4. reloads the page, now the video does not have infinite loop
5. enable infinite loop
6. the video **stills** does not have infinite loop.

Reply:
Hmm, I set it up to switch on/off after a full page reload. I guess binding into the applyChanges was not too hard, now it switches instantly.